Sensor_Temperature.dcm

```diff
@@ -414,6 +414,12 @@ K temperature sensor I2C
 F http://ww1.microchip.com/downloads/en/DeviceDoc/22203b.pdf
 $ENDCMP
 #
+$CMP MCP9844
+D Microchip Technology Digital Temperature Sensor DFN
+K temperature sensor I2C
```

Comment:
"temperature sensor" is redundant, perhaps add TWI as an alias for I²C instead.

Reply:
Okay, I am not quite sure if we should use TWI or I²C when creating parts, as I²C is a registered trademark while TWI is just a synonym.

Reply:
My main point was to not repeat keywords which are already in the description.
So you can remove "temperature sensor" from the keywords as both are already in the description.

I2C and TWI would be useful keywords in my eyes because users might well search I2C or TWI instead of I²C.

Reply:
OK, thans for the clarification

Reply:
Please change the _keyword_ I²C to I2C, otherwise it's redundant again.

Reply:
You're right, I modified it now